packages/block-library/src/group/index.js

```diff
@@ -22,53 +22,49 @@ export { metadata, name };
 export const settings = {
 	icon,
 	example: {
-		innerBlocks: [
-			{
-				name: 'core/paragraph',
-				attributes: {
-					customTextColor: '#cf2e2e',
-					fontSize: 'large',
-					content: __( 'One.' ),
-				},
+		attributes: {
+			layout: {
+				type: 'constrained',
+				justifyContent: 'center',
 			},
-			{
-				name: 'core/paragraph',
-				attributes: {
-					customTextColor: '#ff6900',
-					fontSize: 'large',
-					content: __( 'Two.' ),
+			style: {
+				spacing: {
+					padding: {
+						top: '1em',
+						right: '1em',
+						bottom: '1em',
+						left: '1em',
+					},
 				},
 			},
+		},
+		innerBlocks: [
 			{
-				name: 'core/paragraph',
+				name: 'core/heading',
 				attributes: {
-					customTextColor: '#fcb900',
-					fontSize: 'large',
-					content: __( 'Three.' ),
+					content: __( 'La Mancha' ),
+					textAlign: 'center',
 				},
 			},
 			{
-				name: 'core/paragraph',
+				name: 'core/image',
 				attributes: {
-					customTextColor: '#00d084',
-					fontSize: 'large',
-					content: __( 'Four.' ),
+					url: 'https://s.w.org/images/core/5.3/Glacial_lakes%2C_Bhutan.jpg',
```

Comment:
I don't know if it matters, but maybe we could marry the text with the image?

> Lake Bhutan is a high-altitude lake in Bhutan, known for its pristine beauty, clear waters, and serene surroundings, attracting nature lovers and trekkers.

Or, there are some free use images related to Don Quixote over at https://www.oldbookillustrations.com/search/?q=don+quixote

Reply:
It would be nice if all the blocks' example content had some level of consistency or theme. It's pretty random at the moment 😅